Makefile

```diff
@@ -44,3 +44,11 @@ publish-dev: ## 📤 Publish to the development API Management instance
 		&& export API_MANAGEMENT_SERVICE_NAME=${DEV_APIM_NAME} \
 		&& export AZURE_RESOURCE_GROUP_NAME=${DEV_RESOURCE_GROUP_NAME} \
 		&& publisher
+
+# publish-prod: ## 📤 Publish to the production API Management instance
```

Comment:
@ross-p-smith We obviously do not want to deploy to prod locally, but I've left this commented for now so that we can demonstrate how to pass in a configuration override file to the publisher (line 53). Happy to remove.